spec/spec_helper.rb

```diff
@@ -14,7 +14,7 @@
 
 # Requires supporting ruby files with custom matchers and macros, etc,
 # in spec/support/ and its subdirectories.
-Dir[Rails.root.join("spec/support/**/*.rb")].each {|f| require f}
+Dir[File.join(File.dirname(__FILE__), 'support/**/*.rb')].each { |file| require file }
```

Comment:
Commit message should explain why our usual method doesn't work for extensions and why.